packages/vscode-extension/package.json

```diff
@@ -123,6 +123,11 @@
         "view": "teamsfx-project-and-check-upgradeV3",
         "contents": "%teamstoolkit.viewsWelcome.teamsfx-project-and-check-upgradeV3.content%",
         "enablement": "fx-extension.initialized"
+      },
+      {
+        "view": "teamsfx-feedback",
```

Comment:
Will this view appear in welcome view? We show welcome view when it's not a TeamsFx project.